languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -72131,6 +72131,13 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                     <token postag="VER:INF.*" postag_regexp="yes"/>
                     <example>Magst du noch so schnell rennen können, ein Motorrad holst du nie ein.</example>
                 </antipattern>
+                <antipattern>
+                    <token postag="SENT_START"/>
+                    <token>lässt</token>
```

Comment:
geht das nur mit "lässt"? vlt ist es besser, wenn man das AP allgemeiner macht mit postag="VER:[123]  ?

Sätze wie: "Kauft man immerzu nur ein, (dann) ist man sein Geld schnell los."

Reply:
Ich hatte es mir noch mal angeschaut und in diesen Fällen würde die Regel gar nicht anschlagen:
![image](https://user-images.githubusercontent.com/115984740/209120591-c3c519b8-77e0-4d01-9ac3-3dea1a09f6ec.png)
![image](https://user-images.githubusercontent.com/115984740/209120762-b744ddad-581b-4f7c-a251-c40cd9d59bb7.png)

Reply:
ja, ich habe eben noch mal das pattern angesehen und das findet sowieso nur modalverben, also wüde es "kaufen" nicht finden. "lassen" ist zwar auch als modalverb getaggt, aber der satz aus dem diff schlägt aber auch nicht an...frage wäre dann, ob man das AP überhaupt braucht.

Reply:
Bei mir schlägt es an, wenn ich nicht eingeloggt bin (OS):
![image](https://user-images.githubusercontent.com/115984740/209123746-363d4baf-befe-402d-8899-8a966b8c0409.png)

Reply:
ja, dann ist die frage, warum es in premium nicht anschlägt, irgendwas andere liegt dann darüber...mmmhhh...
was meinst du @AgnesKleinhans ?

Reply:
Weil ich's doch erst nach dem Deploy geändert, also die exception/antipattern hinzugefügt hatte d.h. nach dem Deploy würde es da auch anschlagen.